extensions/wikia/AdEngine3/src/setup.js

```diff
@@ -58,7 +60,7 @@ function setupAdContext(wikiContext, isOptedIn = false, geoRequiresConsent = tru
 
 	context.set('slots', slots.getContext());
 
-	context.set('custom.hiviLeaderboard', isGeoEnabled('wgAdDriverOasisHiviLeaderboardCountries'));
+	context.set('wiki.targeting.hasIncontentPlayer', !context.get('custom.hasFeaturedVideo') && slotInjector.inject('incontent_player'));
```

Comment:
It will put DOM element to context

Reply:
Also I belive we can get rid of injecting the container in the `setup.js` if we go with:
https://github.com/Wikia/app/pull/16842/files#r268066112
https://github.com/Wikia/app/pull/16842/files#r268065979